extension-manifest-v3/src/utils/trackerdb.js

```diff
@@ -9,15 +9,45 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0
  */
 
-import * as engines from './engines.js';
-import { getException } from '/background/exceptions.js';
-
 import { order as categoryOrder } from '@ghostery/ui/categories';
 
+import * as engines from './engines.js';
+import { getException } from '../background/exceptions.js';
+
 let promise = engines.init(engines.TRACKERDB_ENGINE).then(() => {
   promise = null;
 });
 
+export function isCategoryBlockedByDefault(categoryId) {
+  switch (categoryId) {
+    case 'advertising':
+    case 'pornvertising':
+    case 'email':
+    case 'site_analytics':
+    case 'unidentified':
+    case undefined:
+      return true;
+    default:
+      return false;
+  }
+}
+
+export function isTrusted(request, category, exception) {
+  const blockByDefault = isCategoryBlockedByDefault(category);
+
+  let isTrusted = !blockByDefault;
+
+  if (exception) {
```

Comment:
I find it hard to understand this logic.

overwriteStatus is a boolean, from what I see. To me, it is not intuitive what `overwriteStatus=true` or `overwriteStatus=false` means.

The semantic of "isTrusted" might also probably be rather "should not block". (I would expect that "trusted" implies that there was an action from the user before, namely to trust the tracker or the page.)

Reply:
I'm also not super happy with the naming - lets have a discussion.